cea/schemas.yml

```diff
@@ -10,6 +10,7 @@ PVT_metadata_results:
         type: float
         unit: '[m2]'
         values: '{0.0...n}'
+        min: 0.0
```

Comment:
woah. i hope you didn't have to do this by hand...

Reply:
nah once I realized that most of the numerical types didn't have proper range properties, I knew I had to write a small script to do that. Hmm but should I have included that as a utility script? I guess that would be nice.

Reply:
nah, i think you're good: we can expect this to be filled out correctly from now on (since you added the unit test, right?) and won't have to do it again.